src/mesh/Vertex.hpp

```diff
@@ -31,10 +31,20 @@ class Vertex : public PropertyContainer, private boost::noncopyable
   template<typename VECTOR_T>
   void setCoords ( const VECTOR_T& coordinates );
 
+  /// Sets the coordinates of the vertex.
+  /// RValue variant
```

Comment:
Not sure if Doxygen eats and digestes the double `///` correctly. I would just put on a single line. `/// Sets the coordinates of the vertex, rvalue variant`

Reply:
Will merge the lines.